xscen/io.py

```diff
@@ -314,6 +361,11 @@ def save_to_netcdf(
         Spatial dimensions can be generalized as 'X' and 'Y', which will be mapped to the actual grid type's
         dimension names.
         Rechunking is only done on *data* variables sharing dimensions with this argument.
+    bitround : bool or int or dict
+      If not False, float variables are bit-rounded by dropping a certain number of bits from their mantissa, allowing for a much better compression.
+      If an int, this is the number of bits to keep for all float variables.
+      If a dict, a mapping from variable name to the number of bits to keep.
+      If True, the number of bits to keep is guessed based on the variable's name, defaulting to 12, which yields a relative error of 0.012%.
```

Comment:
To be more precise, we should say something like: 
> The **maximum possible absolute relative error** will be "0.0122 ... %", 

as this is an upper bound.  For simplicity, we can drop the "absolute" which can be implicit, but otherwise I would write something like:  
```suggestion
      If True, the number of bits to keep is guessed based on the variable's name, defaulting to 12, which yields a relative error below 0.0125%.
```
or `below 0.013%` if we prefer keeping two significant digits.